mappings/net/minecraft/inventory/Inventory.mapping

```diff
@@ -1,28 +1,28 @@
 CLASS agw net/minecraft/inventory/Inventory
-	METHOD K_ getInvMaxStackAmount ()I
-	METHOD N_ getInvSize ()I
-	METHOD a getInvStack (I)Lbcb;
+	METHOD K_ getSlotCapacity ()I
+	METHOD N_ getInventorySize ()I
+	METHOD a getStack (I)Lbcb;
 		ARG 1 slot
-	METHOD a takeInvStack (II)Lbcb;
+	METHOD a removeStack (II)Lbcb;
```

Comment:
This could be just me, but this change *might* not be the best idea. If I remember correctly, `takeInvStack` is only called when a player manually picks up a stack from the inventory, while the existing `removeInvStack` is used more generally for several different things (hoppers, etc.). It might be better to simply call this `takeStack`, removing the redundant `Inv`.

Reply:
From what I can tell, they have effectively the same semantics, with one just handling splitting logic for removing only a certain amount.

Reply:
On further inspection, the semantics differ more than I thought. In `BasicInventory` for example, the splitting method also notifies change. Everything else seems identical though. Mutation-safe item removal.

Reply:
Worth noting that *only* `BasicInventory` appears to notify change there among the default implementations.